src/go/privileged-service/pkg/port/portproxy.go

```diff
@@ -150,3 +184,8 @@ func portProxyAddArgs(listenPort, listenAddr, connectAddr string) ([]string, err
 		fmt.Sprintf("connectaddress=%s", connectAddr),
 	}, nil
 }
+
+func getHash(portProxy portProxy) string {
+	data := fmt.Sprintf("%v", portProxy)
```

Comment:
Is `%v` going to be consistent in the right way? (Would this include any pointers? Because if it does, then the hash would not match when we want it to.)

Reply:
The `%v` is consistent since the `protProxy` and itsAn sub-structs do not contain any pointers, however, this is not really future-proof, but maybe that's ok since we are retiring the privileged service soon.

An alternative approach is to use gob or json, I will investigate and alternative.